package.json

```diff
@@ -7,6 +7,7 @@
     "@babylonjs/core": "^4.2.0",
     "@babylonjs/loaders": "^4.2.0",
     "@dcl/builder-client": "^1.5.0",
+    "@dcl/content-hash-tree": "^1.1.2",
```

Comment:
the `-lock` file has 44,697 additions :O would you mind checking locally if the diff looks good? GH is not showing me the changes cause it's too large